packages/gatsby/src/schema/__tests__/build-node-types-test.js

```diff
@@ -15,6 +24,19 @@ describe(`build-node-types`, () => {
   }
 
   beforeEach(async () => {
+    const apiRunnerResponse = [
```

Comment:
I think we should use `createPageDependency.mockClear()` - without it last test `should create page dependency` will not fail because page dependency for single `child` (see my other comment in `build-node-types`) was created in earlier test

Reply:
It also be great if test would check if all kinds of node "links" on RootNodeType create page dependency:
 - `parent`
 - `children`
 - `childX` (with single child nodes of given type X)
 - `childrenX` (with multiple child nodes of given type X)

Reply:
Great catch with that mockClear. I added the extra tests as well. Should be good to go